index.css

```diff
@@ -470,3 +476,382 @@ textarea {
   border-radius: 10px;
   margin-bottom: 20px;
 }
+
+/* Desktop version */
+
+.nav-items {
+  display: none;
+}
+
+.border-right {
+  display: none;
+}
+
+.content-image-desktop {
+  display: none;
+}
+
+.project-card .desktop-title {
+  display: none;
+}
+
+.content-desktop {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  /* Nav-bar */
+
+  .menu-icon {
+    display: none;
+  }
+
+  .social li {
+    margin-bottom: 15px;
+  }
+
+  .nav-items ul li {
+    margin-right: 32px;
+  }
+
+  .name {
+    margin-top: 0;
+    margin-left: 50px;
+  }
+
+  .nav-bar {
+    margin-bottom: 20px;
+    margin-top: 20px;
+  }
+
+  .nav-items {
+    margin-right: 70px;
+    display: flex;
+    font-family: "Inter", Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .nav-items ul {
+    display: flex;
+  }
+
+  /* Welcome section */
+
+  .welcome-section {
+    height: 100vh;
+    background-image: url(./images/Illustration-header-desktop-2.png);
+    background-repeat: no-repeat;
+    background-attachment: scroll;
+    background-position: 604px -70px;
+    background-color: #1c1a19;
+    display: flex;
+    flex-direction: row;
+    padding: 109px 24px;
+  }
+
+  .contents {
+    display: flex;
+    flex-direction: column;
+    width: 438px;
+    justify-content: center;
+  }
+
+  .social-links {
+    display: flex;
+    order: -1;
+  }
+
+  .social {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-right: 50px;
+  }
+
+  /* Work section */
+
+  .second-section {
+    margin-left: 50px;
+    margin-right: 30px;
+  }
+
+  .page-title {
+    display: flex;
+    justify-items: baseline;
+    border-bottom: none;
+  }
+
+  .page-title .section-title {
+    margin-right: 14px;
+  }
+
+  .border-right {
+    display: block;
+    width: 794px;
+    height: 0;
+    margin-top: 26px;
+    border-bottom: 1px solid #212121;
+  }
```

Comment:
- [ ] Kindly replace the fixed width `width: 794px;`, with `flex:1;` because it makes your page overflows.

![image](https://user-images.githubusercontent.com/35535130/155629912-267abaa0-0f1c-4636-af2f-07b42a1ac2e7.png)